### DISCUSSION TITLE
Allow sorting releases by (semantic) version, and not time-of-creation

### DISCUSSION THREAD

**Discussion by stdedos** [community/community#4537] [Other Feature Feedback, Questions, & Ideas]
I may have a project that tracks r/next (which releases betas), maint/4.x, maint/3.x (which release (maintenance) _releases_), and sometime I'll carve out maint/5.x (of which I'll release 5.0).

I'd like releases to automatically sort themselves according to "what's latest" and not e.g. like 3.0, 4.0, 3.1, 3.2, 4.1, 5.0, ...

**Comment by svlandeg** [comment]
We'd love to see this for our repo's at Explosion as well. For instance, [spaCy](https://github.com/explosion/spaCy)'s latest release is 3.1.1 but we made a bugfix release 3.0.7 which is now showing up as the "latest" one on Github, which is kind of unfortunate.
